server/base/src/main/java/org/apache/accumulo/server/util/SystemPropUtil.java

```diff
@@ -95,21 +96,36 @@ private static String validateSystemProperty(ServerContext context, SystemPropKe
     // Find the property taking prefix into account
     Property foundProp = null;
     for (Property prop : Property.values()) {
-      if (prop.getType() == PropertyType.PREFIX && property.startsWith(prop.getKey())
+      if ((prop.getType() == PropertyType.PREFIX && property.startsWith(prop.getKey()))
           || prop.getKey().equals(property)) {
         foundProp = prop;
         break;
       }
     }
 
-    if ((foundProp == null || (foundProp.getType() != PropertyType.PREFIX
-        && !foundProp.getType().isValidFormat(value)))) {
+    if (foundProp == null || (foundProp.getType() != PropertyType.PREFIX
```

Comment:
Is this only removing a useless paren?  Seems like that is the case, want to make sure I am not missing something.

Reply:
Yeah, that's the only change